pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -129,16 +161,29 @@ let
       ninja pkg-config
       python3WithPackages perl
       which
-      llvmPackages.bintools
+      buildPackages.${llvmPackages_attrName}.bintools
       bison gperf
     ];
 
+    depsBuildBuild = [
+      buildPlatformLlvmStdenv
+      buildPlatformLlvmStdenv.cc
+      pkgsBuildBuild.pkg-config
+      libuuid
+      libpng  # `.override` used below does not splice
+    ] ++ buildInputs;
```

Comment:
That's weird, Why is it necessary

Reply:
After investigating more closely I think the comment is incorrect.

I think `pkgsBuildBuild.libpng` is needed for something Chromium is doing at build time, not because of anything related to splicing.  I'm verifying this and will remove the comment (or the entire line) once I do.

Reply:
I'm saying `++ buildInputs` is weird

Reply:
Chromium compiles a *ton* of stuff for both the `hostPlatform` and the `buildPlatform` (which, confusingly, it calls "`host`"... argh).  For example, in order to dump out a "quick-start image" for the javascript VM it builds an entire `buildPlatform` version of it!

I could slog through every last dependency and figure out exactly which ones are/aren't needed on the host, but then that list would need to be maintained separately.  It's definitely more than half of them.

Reply:
See ea7c3166548919ba63c204f47598e8522de1c2d3